oap-server/server-cluster-plugin/cluster-zookeeper-plugin/src/main/java/org/apache/skywalking/oap/server/cluster/plugin/zookeeper/ZookeeperCoordinator.java

```diff
@@ -128,4 +134,30 @@ private void initHealthChecker() {
             healthChecker = metricCreator.createHealthCheckerGauge("cluster_zookeeper", MetricsTag.EMPTY_KEY, MetricsTag.EMPTY_VALUE);
         }
     }
+
+    @Override
+    protected void start() {
+        initHealthChecker();
+        serviceCache.addListener(new ZookeeperEventListener());
```

Comment:
Maybe move the line 64 (`this.serviceCache.start();`) here and move this line to the constructor?